server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/main/java/org/apache/james/webadmin/vault/routes/ExportService.java

```diff
@@ -74,7 +74,7 @@ private BlobId zipToBlob(User user, Flux<DeletedMessage> messages) throws IOExce
         try (FileBackedOutputStream fileOutputStream = new FileBackedOutputStream(FileUtils.ONE_MB_BI.intValue())) {
```

Comment:
I think you should create the FileBackedOutputStream with the second parameter resetOnFinalize set to true.
To clear the memory or remove the file when the gc collect the datasource

Reply:
Do we still need these FileBackedOutputStream ? :trollface: